src/eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -88,12 +89,16 @@ def fitlc(eventlabel, ecf_path=None, s4_meta=None, input_meta=None):
 
             t0 = time_pkg.time()
 
-            meta.spec_hw = spec_hw_val
-            meta.bg_hw = bg_hw_val
-
             # Load in the S4 metadata used for this particular aperture pair
-            if meta.data_format == 'eureka':
+            if (meta.data_format == 'eureka' and
+                    (meta.spec_hw != spec_hw_val or meta.bg_hw != bg_hw_val)):
+                meta.spec_hw = spec_hw_val
+                meta.bg_hw = bg_hw_val
                 meta = load_specific_s4_meta_info(meta)
+            elif meta.data_format != 'eureka':
+                meta.spec_hw = spec_hw_val
+                meta.bg_hw = bg_hw_val
```

Comment:
I think I commented on a similar piece of code above about always overwriting `meta.spec_hw` and `meta.bg_hw` and thus taking it out of the for loop.

Reply:
I don't see any such comment - can you clarify what you mean?

Reply:
Ok, never mind.  Please disregard this comment.